.renovaterc.json

```
@@ -10,7 +10,6 @@
     },
     {
       "matchPackagePatterns": [
-        "^com\\.palantir\\.baseline:baseline-error-prone$",
         "^ruby\/setup-ruby$"
```

Given that the removed like worked:
```suggestion
        "^ruby\\/setup-ruby$"
```